labels.mss

```diff
@@ -53,18 +53,34 @@
     text-character-spacing: 2;
   }
 }
-#city[capital='yes'][zoom>=5][zoom<=10],
+
 #city[zoom>=5][zoom<=10],
 #place_low[type='city'][zoom>=7][zoom<=10],
 #place_low[type='town'][zoom>=9][zoom<=10] {
   shield-file: url('icon/place/town.svg');
-  #city[capital='no'],
   [type='city'] {
     shield-file: url('icon/place/city.svg');
+    shield-line-spacing: -2;
+    shield-text-dx: 6;
+  }
+  [type='hc'],[type='ambassade'],[type='capitale'] {
+    shield-fill: @city_text;
+    shield-face-name: @bold;
+    shield-halo-fill: @halo;
+    shield-size: 12;
+    shield-text-dx: 6;
+  }
+  [type='hc'],[type='ambassade'] {
+    shield-file: url('icon/place/ambassade.svg');
+    shield-allow-overlap: true;
```

Comment:
This is quite risky, but I see the point :)
There are a bunch of glitches:
![screenshot-20180216134033-291x193](https://user-images.githubusercontent.com/146023/36308072-52dc2256-131f-11e8-8dd8-70eec15350d4.png)
![screenshot-20180216134025-494x375](https://user-images.githubusercontent.com/146023/36308073-52f66bde-131f-11e8-84e0-bc4738efaaac.png)
![screenshot-20180216133057-497x327](https://user-images.githubusercontent.com/146023/36308074-53101980-131f-11e8-96cf-2c8843abc42f.png)

I guess we can continue a bit by setting direction city by city, but I'm not totally convinced this is not an endless fight ;)
Other option may be to decorrelate image and label with `shield-unlock-image` and let it be more automatic.
But let's continue by manually setting the direction and see how it goes :)

Reply:
oups sorry, here should be my comment : try to launch a make import_city before?

Reply:
ok tou work on forte and me on piano

ldir is not used on forte at the moment, I will add it

Reply:
hmm no, forte use ldir also

Here is what I see

For Piano :

![Piano : capture d ecran 2018-02-16 a 14 22 38](https://user-images.githubusercontent.com/178744/36309527-488d571a-1325-11e8-9812-7f8ec9a6b37b.png)


For Forte : 

![Forte : capture d ecran 2018-02-16 a 14 24 02](https://user-images.githubusercontent.com/178744/36309513-3d45a2ae-1325-11e8-817f-93525034a8bc.png)

I think `make import_city` is missing

Reply:
Yes! sorry, now it's good :)